src/CMakeLists.txt

```diff
@@ -244,35 +213,7 @@ endif()
 set(GFLAGS_USE_TARGET_NAMESPACE TRUE)
 find_package(gflags REQUIRED)
 
-#Add specific definition in case of dependencies build
-
-if(BUILD_curl)
 
-	# ADD CURL_STATICLIB definition 
-	add_definitions(-DCURL_STATICLIB) 
-	
-	if(WIN32 OR WIN64)
-	
-		#Linking not working on windows need to add several system dependency
-		SET(CURL_LIBRARIES ${CURL_LIBRARIES} "Ws2_32.Lib" "Wldap32.Lib" "Crypt32.Lib")
-    else()
-		
-		#zlib
-		find_package(ZLIB REQUIRED)
-	
-		#find_package(CURL) doesn't indicate ZLIB and idn2 dependencies (idn2 used even with --without-libidn option in configure)
-		SET(CURL_LIBRARIES ${CURL_LIBRARIES} ${ZLIB_LIBRARIES} idn2)    
-	endif()	 
-endif()
-
-if(BUILD_OPENSSL)
-	
-	if(WIN32 OR WIN64)
-    	else()
-		#find_package(OpenSSL doesn't indicate -ldl option
-		SET(OPENSSL_LIBRARIES ${OPENSSL_LIBRARIES} "dl")    
-	endif()	 
-endif()
```

Comment:
Done with `include (antares-deps/cmake/additionnal-definitions.cmake)`

Reply:
The definition of `wxWidgets_ROOT_DIR `is done only if `BUILD_wxWidgets `is activated : 

```
if(BUILD_wxWidgets)

  # Define wxWidgets_ROOT_DIR
  set(wxWidgets_ROOT_DIR ${DEPS_INSTALL_DIR})

endif()
```

This is needed to find wxWidgets package.

There is a hack in case of precompiled libraries used so we can use precompiled librairies (see `.github/workflows/linux-BUILD_DEPS.yml ` and `.github/workflows/windows-BUILD_DEPS.yml `)